frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -156,16 +164,23 @@ export const navigationLogic = kea<navigationLogicType>({
                 )
             },
         ],
-        projectNoticeVariant: [
+        projectNoticeVariantWithClosability: [
```

Comment:
Perhaps:

```suggestion
        closableProjectNoticeVariant: [
```
?

No strong feelings here

Reply:
I'm gonna be verbose in this case